ui/js/components/perf/compare.js

```diff
@@ -69,6 +69,75 @@ treeherder.component('phCompareTable', {
     }
 });
 
+treeherder.component('phTrendTable', {
```

Comment:
Just to clarify for me, do you mean have one controller that somehow has two templateURLs i.e. comparetable.html and trendtable.html, or do you mean literally combine the controllers and the actual .htmls also? If so, do I use ng-if's in the .htmls is that the idea? Thanks :)